apps/myaccount/src/components/multi-factor-authentication/authenticators/backup-code-authenticator.tsx

```diff
@@ -0,0 +1,490 @@
+            })
+            .finally(() => {
+                setIsLoading(false);
+            });
+    };
+
+    /**
+     * Download backup codes
+     */
+    const handleDownloadBackupCodes = (): void => {
+        if (backupCodes) {
+            let backupCodeString: string = "";
+
+            for (let i = 0; i < backupCodes.length; i += 2) {
+                if (backupCodes[i + 1] !== undefined) {
+                    backupCodeString +=
+                        [ i + 1 ] +
+                        ". " +
+                        backupCodes[i] +
+                        "\t " +
+                        [ i + 2 ] +
+                        ". " +
+                        backupCodes[i + 1] +
+                        "\n";
+                } else {
+                    backupCodeString += [ i + 1 ] + ". " + backupCodes[i] + "\n";
+                }
+            }
+    
+            const blob = new Blob(
+                [
+                    t(translateKey + "download.heading") + "\n",
+                    t(translateKey + "download.subHeading") + "\n\n",
+                    backupCodeString + "\n",
+                    "(" + profileDetails.username + ")\n\n",
+                    "*" + t(translateKey + "download.info1") + "\n",
+                    "*" + t(translateKey + "download.info2") + new Date()
+                ],
+    
+                { type: "application/json" }
+            );
+    
+            const url = window.URL.createObjectURL(blob);
+            const a = document.createElement("a");
+
+            a.style.display = "none";
+            a.href = url;
+            a.download = "backup_codes.txt";
+            document.body.appendChild(a);
+            a.click();
+            window.URL.revokeObjectURL(url);
+    
+            setIsCodesDownloaded(true);
+            // Sets a success notification.
+            onAlertFired({
+                description: t(translateKey + "notifications.downloadSuccess.genericMessage.description"),
+                level: AlertLevels.SUCCESS,
+                message: t(translateKey + "notifications.downloadSuccess.genericMessage.message")
+            });
+        } else {
+            onAlertFired({
+                description: t(translateKey + "notifications.downloadError.genericError.description"),
+                level: AlertLevels.ERROR,
+                message: t(translateKey + "notifications.downloadError.genericError.message")
+            });
+        } 
+    };
+
+    /**
+     * Copy backup codes
+     */
+    const handleCopyBackupCodes = (event: MouseEvent<HTMLButtonElement>): void => {
+        event.stopPropagation();
+        const contentToBeCopied: string = backupCodes?.join("\n");
+
+        CommonUtils.copyTextToClipboard(contentToBeCopied)
+            .then(()=> setIsCodesCopied(true)); 
+    };
+
+    /**
+     * Render backup codes modal
+     * @returns Backup codes modal
+     */
+    const renderBackupCodeWizard = (): React.ReactElement => {
```

Comment:
Let's explicitly import `ReactElement` from "react".
With JSX Transform in React 17, we can ditch the React import. So let's implicitly import to provision the migration.

```suggestion
    const renderBackupCodeWizard = (): React.ReactElement => {
```